### DISCUSSION TITLE
how to Expand a Distributed MinIO Deployment many time？

### DISCUSSION THREAD

**Discussion by XTUxiongda** [minio/minio#15421] [Q&A] [answered]
the first time , i use
`https://minio{1...4}.example.net:9000/mnt/disk{1...4}/minio`
the second time, i use
`https://minio{1...4}.example.net:9000/mnt/disk{1...4}/minio https://minio{5...12}.example.net:9000/mnt/disk{1...8}/minio`
the third time ,can i use follow?
```
https://minio{1...4}.example.net:9000/mnt/disk{1...4}/minio 
https://minio{5...12}.example.net:9000/mnt/disk{1...8}/minio
https://minio{13...16}.example.net:9000/mnt/disk{1...4}/minio
```
and each time,  the Drive Capacity can use different?

Can I add disks to the original server during capacity expansion

**Comment by klauspost** [comment, marked as answer]
Yes, Yes and Yes

Note that multiple server pools has individual availability guarantees. Generally we recommend to provision further ahead and not do "micro-expansions".
